src/openforms/appointments/templates/appointments/appointment__pdf.html

```diff
@@ -0,0 +1,61 @@
+{% load i18n %}
+<section class="appointment">
+<h2>{% trans "Appointment information" %}</h2>
+
+<table>
```

Comment:
Is there a reason this markup is different from the other markup (and class names) in `submission_report.html`? I would expect this information to just be formatted the same as a normal submission key-value with hardcoded titles since there are no form step names available.